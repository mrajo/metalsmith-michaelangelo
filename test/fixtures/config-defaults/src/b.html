---
title: bar
template: other.html
---