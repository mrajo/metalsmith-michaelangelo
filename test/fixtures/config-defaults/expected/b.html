<h2>bar</h2>